java/code/src/com/redhat/rhn/manager/kickstart/cobbler/CobblerLoginCommand.java

```diff
@@ -49,7 +50,8 @@ public String login(String usernameIn, String passwordIn) {
         args.add(passwordIn);
         String retval = null;
         try {
-            retval = (String) helper.invokeMethod("login", args);
+            byte[] retvalBytes = (byte[]) helper.invokeMethod("login", args);
+            retval = new String(retvalBytes);
```

Comment:
I made a change in cobbler to change the return value from bytes to string. Maybe this was not enough.